modules/ROOT/pages/database/database-connector-get-started.adoc

```diff
@@ -0,0 +1,88 @@
+= Database Connector - Get Started - Mule 4
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:imagesdir: ../../assets/images/
+
+Support Category: Select
+
+Database Connector Version 1.5.4
+
+The Database connector establishes communication between your Mule app and a relational database.
+
+After adding the connector to Studio, the connector and its operations appear in the Mule Palette.
```

Comment:
Hmm, I've never heard of the Mule palette before. Can you confirm that this is correct?

Reply:
Asked Sebastian.

Reply:
Yes, it is still a thing and is correct.